nimble/host/src/ble_gattc.c

```diff
@@ -3345,7 +3345,7 @@ ble_gattc_read_mult_cb_var(struct ble_gattc_proc *proc, int status,
         return 0;
     }
 
-    memset(attr, 0, sizeof(*attr));
+    memset(attr, 0, sizeof(*attr) * proc->read_mult.num_handles);
```

Comment:
`sizeof(attr)`

Reply:
Done

Reply:
Instructions unclear. Now it's properly fixed